apps/judicial-system/backend/src/app/modules/court/test/createCourtCase.spec.ts

```diff
@@ -153,6 +154,7 @@ describe('CourtService - Create court case', () => {
     type
     ${CaseType.CUSTODY}
     ${CaseType.TRAVEL_BAN}
+    ${CaseType.ADMISSION_TO_FACILITY}
```

Comment:
Not sure if this is possible.

Reply:
It should be possible according to Guðný. Are there some guards clauses that check the `CaseType`?